tests/beta/pydantic/schema/test_basic.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Optional
+
+import pydantic
+
+import strawberry
+
+
+def test_basic_type():
+    class User(pydantic.BaseModel):
+        age: int
+        password: Optional[str]
+
+    @strawberry.beta.pydantic.type(User, fields=["age", "password"])
+    class UserType:
+        pass
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def user(self) -> UserType:
+            return UserType(age=1, password="ABC")
+
+    schema = strawberry.Schema(query=Query)
+
+    query = "{ user { age } }"
+
+    result = schema.execute_sync(query)
+
+    assert not result.errors
+    assert result.data["user"]["age"] == 1
+
+
+def test_basic_type_with_list():
+    class User(pydantic.BaseModel):
+        age: int
+        friend_names: List[str]
+
+    @strawberry.beta.pydantic.type(User, fields=["age", "friend_names"])
+    class UserType:
+        pass
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def user(self) -> UserType:
+            return UserType(age=1, friend_names=["A", "B"])
+
+    schema = strawberry.Schema(query=Query)
+
+    query = "{ user { friendNames } }"
+
+    result = schema.execute_sync(query)
+
+    assert not result.errors
+    assert result.data["user"]["friendNames"] == ["A", "B"]
+
+
+def test_basic_type_excluding_fields():
```

Comment:
Semantically this feels backwards. Since the API is strictly about declaring which fields to include, rather than exclude. Perhaps instead have a test that asserts that not providing any fields to include is an error?